test/test-lib-ruby.sh

```diff
@@ -26,7 +26,7 @@ function test_ruby_imagestream() {
     *) echo "Imagestream testing not supported for $OS environment." ; return 0 ;;
   esac
 
-  ct_os_test_image_stream_s2i "${THISDIR}/imagestreams/ruby-${OS}.json" "${IMAGE_NAME}" \
+  ct_os_test_image_stream_s2i "${THISDIR}/imagestreams/ruby-${OS%[0-9]*}.json" "${IMAGE_NAME}" \
```

Comment:
What a strange syntax :)

What does the `%[0-9]*` do? Can you somehow do regex matching in bash string?

Reply:
> What does the `%[0-9]*` do? Can you somehow do regex matching in bash string?

It removes number from end of the string, so `rhel7` becomes `rhel`, to match changes in this PR. (And don't work during Recharge day! :) )

Reply:
Right, I didn't know that. Thanks!